src/constructs/loadbalancing/alb/application-target-group.ts

```diff
@@ -22,6 +22,7 @@ export class GuApplicationTargetGroup extends ApplicationTargetGroup {
     const { app } = props;
 
     const mergedProps = {
+      protocol: ApplicationProtocol.HTTP,
```

Comment:
Minor: should we add a test (and maybe docs?) for this?

Reply:
Aha, yes this was a test that was missed when copying over to the new file structure. Re-added now.